lib/chef/chef_fs/file_system/chef_server/chef_server_root_dir.rb

```diff
@@ -141,7 +142,7 @@ def children
             @children ||= begin
               result = [
                 # /cookbooks
-                CookbooksDir.new("cookbooks", self),
+                versioned_cookbooks ? VersionedCookbooksDir.new("cookbooks", self) : CookbooksDir.new("cookbooks", self),
```

Comment:
Does this mean we can't use versioned and unversioned cookbooks at the same time?